ui/src/views/admin/users/UsersView.vue

```diff
@@ -514,16 +503,24 @@ export default {
       }
     },
     onAccountBlockConfirmation() {
-      // to do: api call for update isBlocked=1 status
-      this.isAccountBlockModalShown = false;
+      const apiData = { email: this.selectedUser.email };
+      adminBlockUser(apiData).then((response) => {
+        this.$notify.toast(response["message"]);
+        this.isAccountBlockModalShown = false;
+        window.location.reload();
+      });
     },
     onAccountBlockCancelation() {
       this.selectedUser.isBlocked = (this.selectedUser.isBlocked)? false : true;
       this.isAccountBlockModalShown = false;
     },
     onAccountUnblockConfirmation() {
-      // to do: api call for update isBlocked=0 status
-      this.isAccountUnblockModalShown = false;
+      const apiData = { email: this.selectedUser.email };
+      adminUnblockUser(apiData).then((response) => {
+        this.$notify.toast(response["message"]);
+        this.isAccountUnblockModalShown = false;
```

Comment:
Also if you are only putting the shown = false in the then block only, then the modal will still be shown when the api catches error. Is this the behaviour wanted?(I cant confirm how it works since my docker is still acting up, sorry)

Reply:
Probably not, I will add the catch block.

Reply:
I have moved out the shown=false so that the modal will be hidden regardless of success or error of the api.